TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Section.js

```diff
@@ -44,7 +47,7 @@ define(
 			if (!this.get('inspector.configuration.' + this.get('_nodeType'))) {
 				this.set('inspector.configuration.' + this.get('_nodeType'), {});
 			}
-			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('group'), this.get('_collapsed'));
+			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('groupName'), this.get('_collapsed'));
```

Comment:
just wanted to check if it was on purpose ;) didn't give the code a testrun.

Overall I very much like to use descriptive variable names in js too :+1: